### PR TITLE
Form sidebar widget

### PR DESCRIPTION
**Pull request by LisaManresa**
- [x] Add the Clubhouse Story ID: [ch24076]

Still waiting on Mark's input on some minor button styles, but otherwise this is complete.

### CODE REVIEW

**Review by paulfalgout — CHANGES_REQUESTED** [review]
whoops left a comment on the commit instead of the review: https://github.com/RoundingWell/care-ops-frontend/commit/503f8c198839b5e6657a5d80217412af69784c8b#commitcomment-46007030

**Review by paulfalgout — CHANGES_REQUESTED** [review]
Ok the blocking PR is in, we can enable tests

File: test/integration/patients/patient/sidebar.js [hunk 1]
```diff
@@ -387,6 +402,16 @@ context('patient sidebar', function() {
     cy
       .get('.alert-box')
       .contains('Engagement settings for this patient could not be found.');
+
+    // TODO: Enable once Patient Form app is complete
```
Comment (paulfalgout) [inline test/integration/patients/patient/sidebar.js]: Still this right?

Comment (LisaManresa) [reply test/integration/patients/patient/sidebar.js]: Sorry @paulfalgout my gmail github filter apparently did not work. This should be ready to go.

**Pull request merged**